### ISSUE TITLE
[Accessibility]: Code intelligence stepper controls

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#35185]
### Audit type

Screen reader navigation

### User journey audit issue

#33516

### Problem description

<img width="884" alt="stepper" src="https://user-images.githubusercontent.com/103087/167503305-4a3e6fde-1961-493a-b89e-d9d735ad33c5.png">

Stepper is a garbage component (or our use of it is garbage). It's extremely difficult to find out what the current value is and what it means.

### Expected behavior

We should figure out what aria settings we are not using or use a different input component entirely.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
Could this just use `<input type="number">`, assuming it's not using that already?

**Comment by umpox** [comment]
Already using type="number", keeping this closed!
